test/fail_compilation/fail3753.d

```diff
@@ -1,4 +1,6 @@
 /*
+DISABLED: dragonflybsd freebsd linux osx
```

Comment:
win32 was also disabled.

Reply:
Was just waiting for all windows tests to run, also to get the actual error message.